src/web/rustdoc.rs

```diff
@@ -447,11 +447,8 @@ pub fn rustdoc_html_server_handler(req: &mut Request) -> IronResult<Response> {
         (target, inner_path.join("/"))
     };
 
-    // If the requested crate version is the most recent, use it to build the url
-    let mut latest_path = if is_latest_version {
-        format!("/{}/{}", name, latest_version)
-    // If the requested version is not the latest, then find the path of the latest version for the `Go to latest` link
-    } else if latest_release.build_status {
+    // Find the path of the latest version for the `Go to latest` and `Permalink` links
+    let mut latest_path = if latest_release.build_status {
         let target = if target.is_empty() {
             &krate.metadata.default_target
         } else {
```

Comment:
This is around the code to modify for "go to latest", I think you can just switch `latest_version` to hard-coding `"latest"` below.

Reply:
I actually need to keep this code, or something like it, because I use it to calculate the target of the `Permalink` URL when the user is on a `/latest/` URL.